diktat-rules/src/main/kotlin/org/cqfn/diktat/ruleset/rules/chapter3/LineLength.kt

```diff
@@ -457,7 +480,7 @@ class LineLength(configRules: List<RulesConfig>) : DiktatRule(
             val hasNewLineBefore: Boolean,
             val indexLastSpace: Int = 0) : LongLineFixableCases()
 
-        class StringTemplate(val node: ASTNode, val delimiterIndex: Int) : LongLineFixableCases()
+        class StringTemplate(val node: ASTNode, val delimiterIndex: Int, val multiLineOffset: Int) : LongLineFixableCases()
```

Comment:
Maybe, make `multiLineOffset` `Int?` to better indicate, that string is single line if it is `null` instead of `0`?

Reply:
I think it will be better when we check if `multiLineOffset == 0`, but when we calculate this variable, it will be worse